Tasks/AzureFileCopyV4/AzureFileCopyRemoteJob.ps1

```diff
@@ -92,9 +93,10 @@ $AzureFileCopyRemoteJob = {
             $additionalArguments = "--recursive --log-level=INFO"
         }
 
-        Write-DetailLogs "##[command] & azcopy copy `"$containerURL*****`" `"$targetPath`" $additionalArguments"
+        $azCopyExeLocation = Join-Path -Path $azCopyFolderPath -ChildPath "AzCopy\AzCopy.exe"
+        Write-DetailLogs "##[command] & `"$azCopyExeLocation`" copy `"$containerURL*****`" `"$targetPath`" $additionalArguments"
```

Comment:
use get-command to get the azcopy.exe command here. then use push-location and pop-location to execute the command